lib/matplotlib/cbook.py

```diff
@@ -783,7 +783,7 @@ def is_sequence_of_strings(obj):
     """
     if not iterable(obj):
         return False
-    if is_string_like(obj):
+    if is_string_like(obj) and not isinstance(obj, np.ndarray):
```

Comment:
Thanks. I'll fix. I now see that Pandas has moved 3 major versions since I encountered this. I need to do a better job keeping all my virtual environments up-to-date.